src/Auth0.OidcClient.UWP/PlatformWebView.cs

```diff
@@ -100,7 +100,7 @@ private async Task<BrowserResult> InvokeAsyncCore(BrowserOptions options, bool s
                 return new BrowserResult
                 {
                     ResultType = BrowserResultType.Success,
-                    Response = wabResult.ResponseData
+                    Response = wabResult.ResponseData?.Replace("\0", string.Empty)
```

Comment:
Add a note here that this is to remove null termination character sometimes present.